homeassistant/components/zigbee/__init__.py

```diff
@@ -272,6 +242,7 @@ class ZigBeeDigitalIn(Entity):
     def __init__(self, hass, config):
```

Comment:
Just pass in the device instead of `hass`.

Reply:
Good idea! 👍